tests/connect_redirect/connect_redirect_tests.cpp

```diff
@@ -467,26 +442,24 @@ get_client_socket(bool dual_stack, _Inout_ client_socket_t** sender_socket)
 }
 
 void
-authorize_test_wrapper(_In_ const struct bpf_object* object, bool dual_stack, _In_ sockaddr_storage& destination)
+authorize_test_wrapper(bool dual_stack, _In_ sockaddr_storage& destination)
 {
     client_socket_t* sender_socket = nullptr;
 
     get_client_socket(dual_stack, &sender_socket);
-    authorize_test(object, sender_socket, destination, dual_stack);
+    authorize_test(sender_socket, destination, dual_stack);
     delete sender_socket;
 }
 
 void
 connect_redirect_test_wrapper(
-    _In_ const struct bpf_object* object,
-    _In_ sockaddr_storage& destination,
-    _In_ sockaddr_storage& proxy,
-    bool dual_stack)
+    _In_ sockaddr_storage& destination, _Inout_ const sockaddr_storage& proxy, bool dual_stack)
```

Comment:
```suggestion
    _Inout_ sockaddr_storage& destination, _In_ const sockaddr_storage& proxy, bool dual_stack)
```
Inout const doesn't make sense.

Reply:
overlook, thanks